integration/Rakefile

```diff
@@ -1,13 +1,38 @@
 require 'rototiller'
 
+namespace :ci do
+  namespace :test do
+
+    desc 'Tests at the component level for the pe-r10k project'
+    task :component => [:check_client_tool_env_vars] do
```

Comment:
I only brought in the :component task, and not :package let me know if that's ok. :)